project/Build.scala

```diff
@@ -155,8 +155,10 @@ object AlgebirdBuild extends Build {
   )
 
   lazy val algebirdTest = module("test").settings(
+    testOptions in Test ++= Seq(Tests.Argument(TestFrameworks.ScalaCheck, "-verbosity", "4"),
+                                Tests.Argument(TestFrameworks.ScalaCheck, "-minSuccessfulTests", "1")),
```

Comment:
TO FIX
By default, scalacheck calls `check` on a property 100 times. However, we want it to call `check` on our property only once, so that the approximate property can run all `n` tests at once and decide whether there were too many failures.

Hopefully there is a way to tell scalacheck how many times to test each property, so that it can run the approximate property tests once, and the normal property tests 100 times.